conditional-grammar.pegjs

```diff
@@ -3,36 +3,36 @@ Expression 'expression'
     return term
   }
 
-Term 'term' 
+Term 'term'
   = BinaryOrExpression
-  
+
 Literal
   = IntegerLiteral
-  
+
 Factor
   = "(" _ term: Term _ ")" { return term; }
   / StatExpression
-  
+
 Stat 'stat'
   = 'hp'
   / 'health'
   / 'atk'
   / 'attack'
-  / 'def'
```

Comment:
Any reason for reordering these?

Reply:
Yes.
The order in which the different stat literals are defined matters.
When 'def' is defined before 'defense' and the input didn't match with 'def', the parser will not even try to match with 'defense'.

This conditional isn't parsed correctly with the current build:
```
:::if{source="Starly" level=20 condition="defense=20+"}
```